docs/core/tools/global-tools-how-to-create.md

```diff
@@ -157,7 +157,7 @@ Next, create a NuGet package for your application.
 dotnet pack
 ```
 
-The `botsay.1.0.0.nupkg` file is created in the folder identified by the `<PackageOutputPath>` XML value from the `botsay.csproj` file, which in this example is the `./nupkg` folder. This makes it easy to install and test. When you want to release a tool publicly, upload it to [https://www.nuget.org](https://www.nuget.org).
+The `botsay.1.0.0.nupkg` file is created in the folder identified by the `<PackageOutputPath>` XML value from the `botsay.csproj` file, which in this example is the `./nupkg` folder. This makes it easy to install and test. When you want to release a tool publicly, upload it to [https://www.nuget.org](https://www.nuget.org), and it will be available through `dotnet tool install` command.
```

Comment:
Given that dotnet install has a global and a local install, I don't think this is 100% accurate. 

Suggestion: instead of that change, add something like this:
```markdown
Once the tool is available on NuGet, developers can perform a user-wide installation of the tool using the `--global` option of the [dotnet tool install](dotnet-tool-install.md) command.
```

Also can you change the nuget,org link to this instead?
`<https://www.nuget.org>`

Reply:
Thanks!
I wrote exactly your words because I'm not good in English and your statement is clear.
But I don't understand your request about `<https://www.nuget.org>`?